composer.json

```diff
@@ -28,7 +28,6 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
         "dflydev/dot-access-data": "^1.1.0",
         "doctrine/common": "^2.5",
-        "doctrine/inflector": "~1.1.0",
```

Comment:
This "dependency" only existed to support PHP 5.6: https://github.com/acquia/blt/pull/1829